boards/nxp/mimxrt1170_evk/board.cmake

```diff
@@ -5,8 +5,10 @@
 #
 
 if(CONFIG_SOC_MIMXRT1176_CM7 OR CONFIG_SECOND_CORE_MCUX)
+ board_runner_args(linkserver "--no-reset")
  board_runner_args(pyocd "--target=mimxrt1170_cm7")
  board_runner_args(jlink "--device=MIMXRT1176xxxA_M7")
+ board_runner_args(jlink "--no-reset")
```

Comment:
These changes seem unrelated?

Reply:
They are related, I am moving this code to be outside of a 
`if(NOT `SYSBUILD)` check